cmd/rofl/deploy.go

```diff
@@ -160,7 +156,8 @@ var (
 				}
 				var offer *roflmarket.Offer
 				for _, of := range offers {
-					if of.Metadata[provider.SchedulerMetadataOfferKey] == machine.Offer {
+					if of.Metadata[provider.SchedulerMetadataOfferKey] == machine.Offer || machine.Offer == "" {
```

Comment:
Since `offers` are not sorted, this will take a random offer? We should probably explicitly sort them before.

Reply:
I'll order them by ID and pick the most fresh one (largest ID).